protocol/bc/transaction.go

```diff
@@ -51,6 +50,7 @@ const (
 
 	// Bit mask for accepted serialization flags.
 	// All other flag bits must be 0.
+	SerTxHash   = 0x0 // this is used only for computing transaction hash - prevout and refdata are replaced with their hashes
```

Comment:
It's used in validation/tx.go: https://github.com/chain/chain/pull/421/files#diff-fb46f84aaf51fdc47bacd0fb932c9f26R208

```
txin.WriteInputCommitment(buf, bc.SerTxHash)
```